README.md

```diff
@@ -179,26 +173,44 @@ Grype lets you define custom output formats, using [Go templates](https://golang
 
 - Specify the path to the template file (`-t ./path/to/custom.template`).
 
-- Grype's template processing uses the same data models as the `json` output format — so if you're wondering what data is available as you author a template, you can use the output from `grype <image> -o json` as a reference.
```

Comment:
I think we should leave this here. This statement is true regardless of any specific examples we provide.